cdisc_rules_engine/operations/get_ct_attribute_values.py

```diff
@@ -0,0 +1,90 @@
+from cdisc_rules_engine.operations.base_operation import BaseOperation
+import pandas as pd
+
+
+class CTAttributeValues(BaseOperation):
+    def _execute_operation(self):
+        """
+        Fetches column order for a given domain from the CDISC library.
+        Returns it as a Series of lists like:
+        0    ["STUDYID", "DOMAIN", ...]
+        1    ["STUDYID", "DOMAIN", ...]
+        2    ["STUDYID", "DOMAIN", ...]
+        ...
+
+        Length of Series is equal to the length of given dataframe.
+        The lists with column names are sorted
+        in accordance to "ordinal" key of library metadata.
+        """
+        return self._get_ct_attribute_values()
+
+    def _get_ct_attribute_values(self):
+        # 1.0 get input variables
+        # -------------------------------------------------------------------
+        cc = "CT_PACKAGE"
+        cv = self.params.ct_attribute
+
+        # 2.0 build codelist from cache
+        # -------------------------------------------------------------------
+        ct_cache = self._get_ct_from_cache(ct_key=cc, ct_val=cv)
+
+        # 3.0 get dataset records
+        # -------------------------------------------------------------------
+        ct_data = self._get_ct_from_dataset(ct_key=cc, ct_val=cv)
+
+        # 4.0 merge the two datasets by CC
+        # -------------------------------------------------------------------
+        cc_key = ct_data[cc].to_list()
+        ct_list = ct_cache[(ct_cache[cc].isin(cc_key))]
+        n = self.params.dataframe.shape[0]
+        result = pd.Series([ct_list[cv].values[0] for _ in range(n)])
+        return result
+
+    def _get_ct_from_cache(self, ct_key: str, ct_val: str):
```

Comment:
Now that we are trying to add docstrings to all functions, would you mind adding them to these two functions? You can follow this guide on how they should be written: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings

Reply:
I did and will sync them to the repo.

Reply:
Docstrings are added.